src/js/components/Avatar/Avatar.js

```diff
@@ -49,12 +52,22 @@ const Avatar = ({
 
   if (typeof src === 'string') {
     return (
-      <StyledAvatar {...avatarProps} {...rest} background={`url(${src})`} />
+      <StyledAvatar
```

Comment:
Rather than repeating this block twice, can we build the `content` using the typeof checks and then put that inside a single `StyledAvatar`?